Translate library/intro

eopXD opened the pull request:
#31

eopXD committed:
translate library/intro

adrianliaw commented:
There are two warnings generated by Sphinx:

```
/.../cpython/Doc/library/intro.rst:15: WARNING: inconsistent term references in translated message. original: [':keyword:`import`'], translated: []
/.../cpython/Doc/library/intro.rst:36: WARNING: inconsistent term references in translated message. original: [':mod:`random`', ':ref:`built-in-funcs`'], translated: []
```

It seems like there are redundant space inside the reference syntax, e.g. <code>:keyword:\`import\`</code> should not become <code>:keyword: \`import\`</code>.

eopXD committed:
error fix

eopXD commented:
@adrianliaw error fixed, sorry for the late reply

adrianliaw committed:
Upgrade to 3.9

adrianliaw committed:
Update readme python version

StevenHsuYL committed:
Translate tutorial/interpreter.po and introduction.po (#67)

* interpreter zh-tw 3.9

Modified from zh-cn 3.9.4 version.

* tutorial/introduction.po

finalize and modify.
zh-tw 3.9

* Delete interpreter.html

* tutorial\interpreter.po and introduction.po

fixed for all review comments.

* tutorial\interpreter.po

final fixation for every "\ ".

* tutorial\interpreter.po

final check for "\ ".

* tutorial\introduction.po

fix "escape".

* Update interpreter.po

fix all "interpreter" and "file" translation.

StevenHsuYL committed:
Translate tutorial/datastructures.po (#71)

StevenHsuYL committed:
Translate tutorial/controlflow.po (#70)

StevenHsuYL committed:
Translate sphinx.po (#75)

StevenHsuYL committed:
Translate tutorial/classes.po

Updated and finished.

StevenHsuYL committed:
Continue tutorial/modules.po (#72)

StevenHsuYL committed:
Translate tutorial/stdlib2.po

only 50% now.

StevenHsuYL committed:
Complete stdlib2.po

Complete translation.

StevenHsuYL committed:
Minor modification (#79)

1. Update fuzzy msgid in modules.po
2. Unify "string literal" in introduction.po

StevenHsuYL committed:
Translate tutorial/inputoutput.po (#74)

* Translate inputoutput.po

1. rst:330 was not updated.
2. Fix a "一" (注音文) in datastructures.po

* Add translator

* Update year of Copyright

* Update datastructures.po

Only change 「」 marks. Thanks!

* Update inputoutput.po

Update msgid of rst:330, 335.
Completed.

StevenHsuYL committed:
Update fuzzy of tutorial/venv.po (#81)

* Update fuzzy of venv.po

Minor modification.

* Update venv.po

Fix the suggested sentence.

dukecat0 committed:
Update copyright.po (#87)

dukecat0 committed:
Update tutorial/whatnow.po (#88)

dukecat0 committed:
Update copyright.po (#91)

StevenHsuYL committed:
Translate license.po

Complete translation of license.po

StevenHsuYL committed:
Update license.po

Fix a rST problem.

StevenHsuYL committed:
Translate intalling/index.po

1. Complete installing/index.po
2. Fix a term "開關參數" in tutorial/module.po

StevenHsuYL committed:
Update license.po

Fix the term "副本".

StevenHsuYL committed:
Update installing/index.

Fix the position of "(switch)".

StevenHsuYL committed:
Translate distributing/index.po

Complete translation.

StevenHsuYL committed:
Translate tutorial/errors.po (#76)

* Translate tutorial/errors.po

Completed and updated.

* Update errors.po

All suggestions were applied.

StevenHsuYL committed:
Update tutorial/stdlib.po (#78)

* Update tutorial/stdlib.po

Update fuzzy and minor modification.

* Update stdlib.po

All suggestions were applied.

StevenHsuYL committed:
Update index pages.

Translate and update several index pages.

StevenHsuYL committed:
Translate extending/index.po

Complete translation.

StevenHsuYL committed:
Update index pages.

"Setup" -> "設置"

StevenHsuYL committed:
Merge branch '3.9' of https://github.com/python/python-docs-zh-tw into index

StevenHsuYL committed:
Update sphinx.po

glossary: "術語表"

StevenHsuYL committed:
Translate tutorial/interactive.po (#83)

* Translate interactive.po

1. Translate tutorial/interactive.po
2. Update fuzzy of tutorial/whatnow.po

* Update interactive.po

All suggestions were applied.

* merge conflict solved.

with minor revision.

dukecat0 committed:
Translate tutorial/appendix.po (#84)

* Translate tutorial/appendix.po

* Translate tutorial/appendix.po

* Update Project-Id-Version

* Update appendix.po

* Update tutorial/appendix.po

* Update tutorial/appendix.po

StevenHsuYL committed:
Update tutorial/stdlib.po (#97)

* Update tutorial/stdlib.po

Update fuzzy and minor modification.

* Update stdlib.po

All suggestions were applied.

* Update tutorial/stdlib.po

Apply all suggestion in #78

* Apply suggestions from review.

Apply all suggestions. Thanks!

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

* Update tutorial/stdlib.po

Procedure: "程序"

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

StevenHsuYL committed:
Merge branch '3.9' of https://github.com/python/python-docs-zh-tw into index

StevenHsuYL committed:
Update sphinx.po

Update the welcome message in sphinx.po

bonzoyang committed:
secrets translation v 1.0 (#66)

StevenHsuYL committed:
Apply suggestions from code review

All suggestions were applied.

Co-authored-by: Benson Chen <bc842017@gmail.com>

StevenHsuYL committed:
Update stdlib2.po

Fix a missing white space.

mattwang44 committed:
feat: automation with github action (#102)

* Pulling latest source strings from CPython daily and make the change as a PR (only support branch 3.9).
  * Maintainer should resolve the fuzzy entries before merging it.
  * If the PR for this task is existed, changes will be forced-push to the existing branch (checkout peter-evans/create-pull-request@v3).
* Launch build flow during a pull request is opened (supported base branch: 3.7, 3.8, & 3.9).
* Deploy to github page if any change is pushed to main branch (current: 3.9).

github-actions[bot] committed:
sync with cpython 9ad2184d (#104)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

adrianliaw committed:
Revert "sync with cpython 9ad2184d (#104)" (#106)

This reverts commit d54eaf6bf4e2e4f139eeb809b3ef6dcba359c305.

mattwang44 committed:
chore: insert `Language` header entry when merging from CPython doc

mattwang44 committed:
chore(doc): update the link for current translated doc

mattwang44 committed:
chore: send cpython-syncing PR as a draft

mattwang44 committed:
chore: remove reviewer for cpython-syncing PR

Remove the configuration since it will assign accounts that is in python core team but not in pydoc-zh-tw as a reviewer.

mattwang44 committed:
chore: insert github app token for triggering ci on automation PR

mattwang44 committed:
Merge pull request #108 from mattwang44/improve-github-action

Improve GitHub action workflows

mattwang44 committed:
fix(sphinx): update based on changes from cpython syncing

mattwang44 committed:
fix(tutorial/controlflow): update based on changes from cpython syncing

mattwang44 committed:
fix(copyright): update based on changes from cpython syncing

mattwang44 committed:
fix(faq/general): update based on changes from cpython syncing

mattwang44 committed:
fix(faq/programming): update based on changes from cpython syncing

mattwang44 committed:
fix(howto/descriptor): update based on changes from cpython syncing

mattwang44 committed:
fix(library/array): update based on changes from cpython syncing

mattwang44 committed:
fix(c-api/dict): update based on changes from cpython syncing

github-actions[bot] committed:
sync with cpython 1b02e29f

mattwang44 committed:
fix(action): allow merging cpython on existing branch

mattwang44 committed:
Merge pull request #110 from mattwang44/fix/sync-workflow/edit-on-branch

fix(action): allow merging cpython on existing branch

StevenHsuYL committed:
Translate tutorial/floatingpoint.po (#89)

StevenHsuYL committed:
Translate bugs.po (#93)

1. Complete bugs.po
2. Update year in copyright.po

Co-authored-by: setmao <set.mao.tw@gmail.com>

mattwang44 committed:
Merge branch '3.9' into cron/sync/3.9

github-actions[bot] committed:
sync with cpython 4ce55cce

github-actions[bot] committed:
sync with cpython 9443ce4e

mattwang44 committed:
fix(tutorial/floatingpoint): update based on changes from cpython syncing

adrianliaw committed:
Merge pull request #105 from python/cron/sync/3.9

mattwang44 committed:
Merge branch '3.9' into classes

mattwang44 committed:
Merge branch '3.9' into distributing

mattwang44 committed:
Merge branch '3.9' into index

mattwang44 committed:
Merge branch '3.9' into installing

StevenHsuYL committed:
Update classes.po

Unify "reference" to "參照"。Apply review suggestions.

StevenHsuYL committed:
Update classes.po

Fix a typo.

StevenHsuYL committed:
Update sphinx.po

Sync with zh-tw branch 3.9

adrianliaw committed:
Merge pull request #80 from StevenHsuYL/stdlib2

Translate tutorial/stdlib2.po

StevenHsuYL committed:
Merge branch '3.9' of https://github.com/python/python-docs-zh-tw into license

adrianliaw committed:
Merge pull request #77 from StevenHsuYL/classes

Continue tutorial/classes.po

StevenHsuYL committed:
Update sphinx.po

"glossary" to "術語表", sync with glossary.po

StevenHsuYL committed:
Update distributing/index.po

Apply review suggestions.

StevenHsuYL committed:
Update distributing/index.po

PyPI: "Python Package Index", sync with cpython. bpo-45341.

StevenHsuYL committed:
Update installing/index.po

PyPI: "Python Package Index". Sync with cpython.

StevenHsuYL committed:
Update installing/index.po

Sync with cpython, "macOS".

StevenHsuYL committed:
Update installing/index.po

add "(switch)" for "開關參數".

StevenHsuYL committed:
Update license.po

sync with cpython, "macOS".

StevenHsuYL committed:
Update tutorial/appetite.po

sync with cpython, "macOS".

StevenHsuYL committed:
Update tutorial/modules.po

sync with cpython.

mattwang44 committed:
Merge pull request #96 from StevenHsuYL/distributing

pydoc-zh-tw[bot] committed:
Sync with CPython 3.9 (#112)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>

mattwang44 committed:
Merge branch '3.9' into installing

mattwang44 committed:
Merge branch '3.9' into index

mattwang44 committed:
Merge branch '3.9' into license

jordanSu committed:
Translate library/text.po

grimmerk committed:
docs: improve wording and add introduction to pot file

jordanSu committed:
update library/text.po according to suggestions

mattwang44 committed:
Merge pull request #122 from grimmer0125/improve_readme

mattwang44 committed:
fix(README): broken anchor due to modified title

mattwang44 committed:
Merge pull request #120 from jordanSu/library-text

josix committed:
Merge pull request #123 from python/fix/readme-broken-anchor

fix(README): broken anchor due to modified title

grimmerk committed:
docs: add more detailed introduction to readme

StevenHsuYL committed:
Apply suggestions from code review

Apply the suggestion.

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

StevenHsuYL committed:
Apply suggestions from code review

Apply all suggestions.

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

StevenHsuYL committed:
Apply suggestions from code review

Apply part of suggestions.
Pending further reply.

Co-authored-by: Wei-Hsiang (Matt) Wang <mattwang44@gmail.com>

mattwang44 committed:
Merge pull request #95 from StevenHsuYL/installing

mattwang44 committed:
Merge pull request #99 from StevenHsuYL/index

mattwang44 committed:
Merge pull request #94 from StevenHsuYL/license

pydoc-zh-tw[bot] committed:
Sync with CPython 3.9 (#121)

Co-authored-by: github-actions[bot] <github-actions[bot]@users.noreply.github.com>
Co-authored-by: Matt.Wang <mattwang44@gmail.com>

grimmerk committed:
docs: update the sentences about searching fuzzy entries

mattwang44 committed:
fix(README): wrong Telegram invitation link

grimmerk committed:
docs: correct some typo

grimmerk committed:
docs: add back some heapq translations which were accidentally deleted

grimmerk committed:
Update README.rst

Co-authored-by: Josix <wilson8507@gmail.com>

josix committed:
Merge pull request #124 from grimmer0125/organize_readme

docs: organize readme

grimmerk committed:
fix(README): a translation typo

josix committed:
Merge pull request #128 from grimmer0125/fix_readme_typo

fix(README): a translation typo

mattwang44 committed:
chore: update for Python 3.9

mattwang44 commented:
A PR tends to be merged to `3.9` branch is opened (see #131).
